### PR TITLE
<StyledNestableList/> UI fixes, a11ty 

### PR DESCRIPTION
**Pull request by AndriiTsarenko**
In this PR was fixed most issues from former PR https://github.com/wix/wix-style-react/pull/6231
Except:

- Animations
- The cursor should be grabbing type once the item is picked up and dragged. (it's not a trivial issue https://github.com/react-dnd/react-dnd/issues/325)

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6416

File: src/BadgeSelectItem/BadgeSelectItem.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ BadgeSelectItem.propTypes = {
   /** The secondary text to be rendered within the badge. */
   subtitle: PropTypes.string,
   /** The badge's skin. See the <Badge/> story for the full list of available skins. */
-  skin: PropTypes.string.isRequired,
+  skin: PropTypes.string,
```
Comment (AndriiTsarenko) [inline src/BadgeSelectItem/BadgeSelectItem.js:50]: This triggered error in storybook

File: src/NestableList/Container.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ import React, { Component } from 'react';
 import classNames from 'classnames';
 import Item from './Item';
 import { getDepth } from './utils';
-import Collapse from '../Collapse';
```
Comment (AndriiTsarenko) [inline src/NestableList/Container.js:5]: Collapse is buggy for now so I don't use collapse.

File: src/NestableList/Container.js [hunk 3]
```diff
@@ -33,36 +33,53 @@ class Container extends Component {
...
           const children = item[childrenProperty];
+          const hasChildren = children && children.length;
+          const isLastItem = items.length - 1 === i;
+          const actionButton = renderAction({
+            siblings: items,
+            item,
+            veryLastItem: isLastItem && isParentLastItem,
+            depth: treeDepth,
+          });
+          const veryLastItem =
```
Comment (AndriiTsarenko) [inline src/NestableList/Container.js:44]: This is needed to hide the bottom border on the last item or the action button

File: src/NestableList/Container.js [hunk 4]
```diff
@@ -33,36 +33,53 @@ class Container extends Component {
...
-                  treeDepth={treeDepth + 1}
-                />
-              ) : null}
-              {!item.isCollapsed &&
-                renderAction({
-                  siblings: items,
-                  item,
-                  depth: treeDepth,
-                })}
+              {!item.isCollapsed && (
```
Comment (AndriiTsarenko) [inline src/NestableList/Container.js:63]: Grouped children a little

**Review by justinara — CHANGES_REQUESTED** [review]
Nice job! I see it's keyboard accessible now! But I noticed a slight difference from the specs - before being able to reorder item, you should "pick it up" by pressing ENTER key down. Once picked up item get the same drop shadow and a background colour. You can check the full flow in the specs: https://zpl.io/bJJm8rD

File: src/StyledNestableList/StyledNestableList.js [hunk 5]
```diff
@@ -114,11 +177,118 @@ class StyledNestableList extends React.PureComponent {
...
+
+    if (!this.state.movedItem) {
+      return;
+    }
+
+    switch (e.key) {
+      default:
+      case enter:
+        this.releaseItems();
+        break;
```
Comment (EgoziE) [inline src/StyledNestableList/StyledNestableList.js:208]: Just for readability, default should be the last case.

**Review by EgoziE — CHANGES_REQUESTED** [review]
Please add to all non public functions an underscore.

**Pull request merged**